bibliopixel/cli/version.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""
+Prints version information
+"""
+
+from bibliopixel import __name__, __version__, __status__, __copyright__
+from bibliopixel.cli import Command
+
+
+class Version(Command):
+    name = "version"
+    description = "Prints the version, copyright, and release status"
+
+    def main(self):
+        print(__name__ + " "
```

Comment:
1. Use some form of formatted print here.

2. We never use raw `print` in our code - we only use it for debugging statements.  There's a simple reason for this - we have our flake8 set to flag raw `print` statements to prevent us from checking in code with debugging statements (something we otherwise did pretty reliably about 2% of the time before).

Use `log.printer` instead.